gatherers/manifest.js

```diff
@@ -47,37 +47,14 @@ class Manifest extends Gather {
   static gather(options) {
     const driver = options.driver;
 
-    return driver.sendCommand('DOM.getDocument')
-        .then(result => result.root.nodeId)
-        .then(nodeId => driver.sendCommand('DOM.querySelector', {
-          nodeId: nodeId,
-          selector: 'link[rel="manifest"]'
-        }))
-        .then(manifestNode => manifestNode.nodeId)
-        .then(manifestNodeId => {
-          if (manifestNodeId === 0) {
-            return '';
-          }
-
-          return driver.sendCommand('DOM.getAttributes', {
-            nodeId: manifestNodeId
-          })
-          .then(manifestAttributes => manifestAttributes.attributes)
-          .then(attributes => {
-            const hrefIndex = attributes.indexOf('href');
-            if (hrefIndex === -1) {
-              return '';
-            }
-
-            return attributes[hrefIndex + 1];
-          })
-          .then(manifestURL => Manifest._loadFromURL(options, manifestURL));
-        })
-        .then(manifestContent => {
-          return {
-            manifest: manifestParser(manifestContent).value
-          };
-        });
+    return driver.querySelector('head link[rel="manifest"]')
+      .then(node => node.getAttribute('href'))
+      .then(manifestURL => Manifest._loadFromURL(options, manifestURL))
```

Comment:
we should document (or file an issue if we want to change it) that since the page isn't fetching the manifest, this might not be the same manifest the page would get (e.g. if the user needs to be logged in)